cg/store/crud/read.py

```diff
@@ -471,11 +471,15 @@ def get_flow_cells_by_case(self, case: Case) -> list[Flowcell] | None:
             case=case,
         ).all()
 
-    def get_samples_from_flow_cell(self, flow_cell_id: str) -> list[Sample] | None:
-        """Return samples present on flow cell."""
-        flow_cell: Flowcell = self.get_flow_cell_by_name(flow_cell_name=flow_cell_id)
-        if flow_cell:
-            return flow_cell.samples
+    def get_samples_by_illumina_flow_cell_internal_id(
+        self, flow_cell_id: str
+    ) -> list[Sample] | None:
```

Comment:
Isn't the "by" part clear by what you pass as to the function?
```suggestion
    def get_samples_by_illumina_flow_cell(
        self, flow_cell_interna_id: str
    ) -> list[Sample] | None:
```

Reply:
I will include your suggestion in the following PR